.golangci.yml

```diff
@@ -1,11 +1,61 @@
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    # - dogsled (TODO: write rational for disabling here, brought over from Makefile)
+    # - dupl (TODO: write rational for disabling here, brought over from Makefile)
+    - errcheck
+    - exhaustive
+    - funlen
+    # - gochecknoglobals (TODO: write rational for disabling here, brought over from Makefile)
+    # - gochecknoinits (TODO: write rational for disabling here, brought over from Makefile)
+    - gocognit
+    # - goconst (TODO: write rational for disabling here, brought over from Makefile)
+    - gocritic
+    - gocyclo
+    - godot
+    - godox
+    # - goerr113 (TODO: expose errors proerply, once/if approaching library use, as oposed to current cli-only use)
+    - gofmt
+    - goimports
+    - golint
+    # - gomnd (TODO: write rational for disabling here, brought over from Makefile)
```

Comment:
Our tests contain hard-coded test data that wouldn't make sense to extract into constants.